charts/airflow-operator/templates/configmap/runtime-versions.yaml

```
@@ -0,0 +1,17 @@
+##############################
+# Runtime Version ConfigMap ##
+##############################
+{{ if and .Values.airgapped .Values.runtimeVersions.versionsJson .Values.global.airflowOperator.enabled (not .Values.runtimeVersions.configMapName) -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-runtime-version-config
+  namespace: '{{ .Release.Namespace }}'
```

nit: This should probably use `quote`, but I bet we do this in a lot of other places too so we don't need to worry about it right now.

```suggestion
  namespace: {{ .Release.Namespace | quote }}
```